interface/expand_contract_js.php

```diff
@@ -0,0 +1,51 @@
+<?php
+  /**
+ * expand contract jquery script
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+// ensure that $user_settings_php_path, $arr_files_php variables are set in the script calling this script
+?>
+$( document ).ready(function() {
+    $('.expand_contract').click(function() {
+        var elementTitle = $(this).prop('title');
+        var contractTitle = '<?php echo xla('Click to Contract and set to henceforth open in Centered mode'); ?>';
+        var expandTitle = '<?php echo xla('Click to Expand and set to henceforth open in Expanded mode'); ?>';
+        //var userSettingsPath = '<?php echo $GLOBALS['webroot'] . "/library/ajax/user_settings.php"; ?>';
+        var arrFiles = [];
+        
+        <?php
+        if (strpos($arr_files_string, ",")) {?>
+              arrFiles = '<?php echo attr($arr_files_string);?>'.split(",");
+            <?php
+        } else {?>
+             arrFiles = ['<?php echo attr($arr_files_string);?>'];
+            <?php
+        } ?>
+        
+        if (elementTitle == contractTitle) {
+            elementTitle = expandTitle;
+            $(this).toggleClass('fa-expand fa-compress');
+            $('.expandable').toggleClass('container container-fluid');
+            if ($(arrFiles).length) {
+                $.each(arrFiles, function (index, value) {
+                  $.post( '<?php echo $user_settings_php_path; ?>', { target: arrFiles[index].trim(), setting: 0 });
+                });
+            } 
+        } else if (elementTitle == expandTitle) {
+            elementTitle = contractTitle;
+            $(this).toggleClass('fa-compress fa-expand');
+            $('.expandable').toggleClass('container-fluid container');
+            if ($(arrFiles).length) {
+                $.each(arrFiles, function (index, value) {
+                  $.post( '<?php echo $user_settings_php_path; ?>', { target: arrFiles[index].trim(), setting: 1 });
```

Comment:
See here for an example of how to call this script with a relative path:
https://github.com/openemr/openemr/blob/master/interface/main/left_nav.php#L1890